app/controllers/admin_user_controller.rb

```diff
@@ -88,12 +88,24 @@ def update
     end
   end
 
+  def active
+    @title = 'Active users'
+    @base_scope = User.active
```

Comment:
Not caused by this PR, but adding this prompted me to create https://github.com/mysociety/alaveteli/issues/6924. No problem for getting this merged, but just noting for posterity.